src/Display/Modal/Modal.tsx

```diff
@@ -37,6 +37,7 @@ export const Modal: FC<Props> = ({
   className,
   hideContentArea,
   centering,
+  fullscreen,
```

Comment:
Would be nice to give this an explicit default.

Reply:
Okay, added it in `defaultProps`.